config_bucket.go

```diff
@@ -29,15 +29,20 @@ func (b BucketType) validate() error {
 // BucketsConfig represents the admin config section for interally-owned
 // Evergreen data bucket storage.
 type BucketsConfig struct {
+	// LogBucket is the
```

Comment:
flagging this line

Reply:
Oops fixed! I didn't save the other half, thank you for rereviewing!